docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -849,20 +849,27 @@ Default stratifications to all observers should include scenario and input draw.
     - 
     - 
   * - 7.0
+    - Wave I neonatal probiotics with scale-up scenarios 
```

Comment:
I think we should specifically note the change we are making from 6.0 (preterm birth eligibility) and link the PR where you made this change so we can have a record of the difference